spec/models/lockstep/magic_link_spec.rb

```diff
@@ -5,15 +5,15 @@
   context 'when generating' do
     it 'should return 401 since api key not set' do
       VCR.use_cassette("models/lockstep/magic_links/unauthorized") do
-        response =  Lockstep::MagicLink.generate('user@email.com', 720, "dummy-app-id", "dummy-user-role")
+        response =  Lockstep::MagicLink.generate('user@email.com', 720, "dummy-app-id", "dummy-user-role", nil)
```

Comment:
wont this throw error actually if the company_id is nil? can we have a valid company_id and regenerate the fixtures with company_id being sent actually?

Reply:
Changes done